py/torch_tensorrt/_compile.py

```diff
@@ -91,8 +97,16 @@ def _get_target_ir(module_type: _ModuleType, ir: str) -> _IRType:
                     "Input graph is a Torchscript module but the ir provided is default (dynamo). Please set ir=torchscript to suppress the warning. Compiling the module with ir=torchscript",
                 )
                 return _IRType.ts
+            elif module_is_exportable:
+                raise ValueError(
+                    "Input graph is an ExportedProgram which is not currently supported. Please provide torch.nn.Module or torch.fx.GraphModule as input."
+                )
```

Comment:
Could add a note that support is coming soon, to differentiate this input type from completely unknown IR.